app/javascript/mastodon/features/ui/index.js

```diff
@@ -176,8 +181,13 @@ export default class UI extends React.PureComponent {
     const { width, draggingOver } = this.state;
     const { children } = this.props;
 
+    let className = 'ui';
+
+    if (this.props.systemFontUi)
+      className += ' has-system-font';
```

Comment:
this would be more cleanly implemented as an array, using classes.join(" ") at the calling site.